### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Liubomyr19**
[DEMO LINK](https://liubomyr19.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://liubomyr19.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Liubomyr19.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Liubomyr19.github.io/layout_moyo-header/report/html_report/)
```
Comment (loralevitska) [inline readme.md:4]: Add this links to PR description 
image

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,51 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="example">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header id="home" class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="logo"
+        class="logo__image" crossorigin="example">
```
Comment (loralevitska) [inline src/index.html:18]: Fix your code style 
image

Check yourself with [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="example">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header id="home" class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="logo"
```
Comment (loralevitska) [inline src/index.html:17]: Consider to use more meaningful alt name
```suggestion
        <img src="images/logo.png" alt="Moyo logo"
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,190 @@
...
+  width: 65%;
+  border-radius: 3px;
+  background-color: #00acdc;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  transform: translateX(10px);
+}
+
+@media (max-width: 1024px) {
```
Comment (loralevitska) [inline src/style.css:64]: remove redundant media 
`justify-content: space-between` property helps for this point

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job! There are things that should be fixed:

- fix paddings and add margins between items. I guess it will help pass you tests
image
image

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,52 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="example">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header id="home" class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="Moyo logo"
+        class="logo__image" crossorigin="example">
```
Comment (DarkMistyRoom) [inline src/index.html:19]:
```suggestion
        <img 
          src="images/logo.png" 
          alt="Moyo logo"
          class="logo__image" 
          crossorigin="example"
        >
```

Comment (DarkMistyRoom) [reply src/index.html:19]: one attribute per line. fix it everywhere

File: src/index.html [hunk 6]
```diff
@@ -1,14 +1,52 @@
...
+      </a>
+      <nav class="nav">
```
Comment (DarkMistyRoom) [inline src/index.html:21]: Add empty lines between multiline sibling blocks of HTML

Comment (DarkMistyRoom) [reply src/index.html:21]: fix it everywhere

File: src/style.css [hunk 7]
```diff
@@ -1 +1,186 @@
...
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  margin-right: -20px;
```
Comment (DarkMistyRoom) [inline src/style.css:25]: looks redundant

File: src/style.css [hunk 8]
```diff
@@ -1 +1,186 @@
...
+  position: relative;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:49]: combine these selectors

File: src/style.css [hunk 9]
```diff
@@ -1 +1,186 @@
...
+  font-size: 12px;
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
```
Comment (DarkMistyRoom) [inline src/style.css:47]:
```suggestion
.is-active {
```

Comment (DarkMistyRoom) [reply src/style.css:47]: don't increase specificity

File: src/style.css [hunk 10]
```diff
@@ -1 +1,186 @@
...
+  color: #00acdc;
+}
+
+.nav__link.is-active:after {
```
Comment (DarkMistyRoom) [inline src/style.css:51]:
```suggestion
.is-active:after {
```

File: src/style.css [hunk 11]
```diff
@@ -1 +1,186 @@
...
+    display: block;
+    height: 4px;
+    width: 100%;
+    border-radius: 3px;
+    background-color: #00acdc;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+}
```
Comment (DarkMistyRoom) [inline src/style.css:186]: no need to use media here. your styles should work properly for screens

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Pull request closed without merging**